BCnEnc.Net/Encoder/Bc5BlockEncoder.cs

```diff
@@ -8,7 +8,7 @@ namespace BCnEncoder.Encoder
 {
 	internal class Bc5BlockEncoder : BaseBcBlockEncoder<Bc5Block>
 	{
-		protected override Bc5Block EncodeBlock(RawBlock4X4Rgba32 block, CompressionQuality quality) {
+		public override Bc5Block EncodeBlock(RawBlock4X4Rgba32 block, CompressionQuality quality) {
```

Comment:
Could change bc5blocks to be just two bc4blocks, since it became more generic. And bc5encoder can just call bc4encoder twice.

Reply:
True, I will change that.